imap/imapd.c

```diff
@@ -13933,9 +13940,9 @@ static int list_data_remote(struct backend *be, char *tag,
                 /* print status items */
                 const char *status_items[] = {
                     /* XXX  MUST be in same order as STATUS_* bitmask */
-                    "messages", "recent", "uidnext", "uidvalidity",
-                    "unseen", "highestmodseq", "size", "mailboxid",
-                    "deleted", "deleted-storage", "",
+                    "messages", "recent", "uidnext", "uidvalidity", "unseen",
+                    "highestmodseq", "appendlimit", "size", "mailboxid",
+                    "deleted", "deleted-storage", "", "", "",
```

Comment:
what's this "empty string" stuff?  It's going to cause:

```
                    prot_printf(be->out, "%c%s", c, status_items[j]);
                    c = ' ';
```

Which will mean if any of these appear there'll be multiple spaces in the result, which probably breaks parsing.

Reply:
@brong The strings need to match the STATUS_* bits and we put the non-standard ones as the last bits, have 3 open bits (11-13).  See imapd.h

Reply:
I confirmed that the placeholders are required when trying to proxy 
`LIST "" % RETURN (STATUS (CREATEDMODSEQ))`

I added a comment next to the placeholders